vuepress/docs/next/tutorials/devops/installation/tanzu/tanzu-install.md

```diff
@@ -0,0 +1,174 @@
+
+1. Update the vsphere-csi-controller configuration.
+```
+kubectl patch deployment -n kube-system vsphere-csi-controller --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/4/args/-", "value": "--default-fstype=ext4"}]'
+```
+2. Delete the vsphere-csi-controller pod.
+```
+kubectl delete pod -n kube-system -l app=vsphere-csi-controller
+```
+Deleting the pod causes it to be recreated with the new configuration.
+
+## Deploy the NGINX Ingress Controller
+
+1. Deploy NGINX
+
+```
+helm install --name ingress-nginx ingress-nginx/ingress-nginx
+```
+
+2. Watch for the nginx pod to be in a status of Running. For example
+
+```
+kubectl get pods
+```
+
+```
+NAME                                            READY   STATUS    RESTARTS   AGE
+pod/ingress-nginx-controller-66dc9984d8-z5x46   1/1     Running   0          116m
+pod/kuard-86664f98c9-7kqb5                      1/1     Running   0          74m
+pod/kuard-86664f98c9-bx4zz                      1/1     Running   0          74m
+pod/kuard-86664f98c9-fs27d                      1/1     Running   0          74m
+```
+
+3. Note the value of the `EXTERNAL-IP` for the nginx controller
+
+```
+kubectl get services
+```
+
+```
+NAME                                         TYPE           CLUSTER-IP       EXTERNAL-IP      PORT(S)                      AGE
+service/ingress-nginx-controller             LoadBalancer   100.66.153.199   192.168.100.50   80:30575/TCP,443:31235/TCP   116m
+service/ingress-nginx-controller-admission   ClusterIP      100.70.59.21     <none>           443/TCP                      116m
+service/kuard                                ClusterIP      100.64.49.203    <none>           80/TCP                       74m
+service/kubernetes                           ClusterIP      100.64.0.1       <none>           443/TCP                      42h
+````
+
+In the example above the EXTERNAL-IP used in the setup below is 192.168.100.50.
+
+
+## Setup and Deploy
+
+1. Create a namespace for your Entando deployment
+
+```
+kubectl create namespace entando
+```
+
+2. Download and unpack the entando-helm-quickstart:
+
+```
+curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.3.2.tar.gz | tar xvz
+```
+3. Deploy the Entando Kubernetes custom resources and configuration
+```
+kubectl apply -n entando -f https://raw.githubusercontent.com/entando-k8s/entando-k8s-operator-bundle/v6.3.2/manifests/k8s-116-and-later/namespace-scoped-deployment/all-in-one.yaml
+```
+
+4. In the entando-helm-quickstart edit this file `sample-configmaps/entando-operator-config.yaml`
+5. Add these properties to the file (taking note of correct yaml spacing):
+
+```
+  entando.requires.filesystem.group.override: "true"
+  entando.ingress.class: "nginx"
+```
+
+6. Find this property in the file `entando.default.routing.suffix:`
+7. Change the value to `<your nginx ip>.nip.io`. For example, `entando.default.routing.suffix: 35.232.231.65.nip.io`
+
+:::note
```

Comment:
This isn't rendering correctly for me so I think the syntax is off.  I think it should be "::: tip"

Reply:
fixed